### PR TITLE
ENQT-130: fixed webDynamicForm plugin dependencies

### PR DESCRIPTION
**Pull request by Rguerrini**

### CODE REVIEW

**Comment by entando-service** [comment]
Can one of the admins verify this patch?

File: admin-console/src/main/webapp/WEB-INF/apsadmin/jsp/entity/include/entity-type-list.jsp [hunk 1]
```diff
@@ -3,204 +3,26 @@
...
-        <s:text name="%{'title.' + entityManagerName + '.management'}"/>
-        <span class="pull-right"> <a tabindex="0" role="button"
-                                     data-toggle="popover" data-trigger="focus" data-html="true" title=""
-                                     data-content="TO be inserted" data-placement="left"
-                                     data-original-title=""><i class="fa fa-question-circle-o"
-                                                               aria-hidden="true"></i></a>
+<ol class="breadcrumb page-tabs-header breadcrumb-position">
+    <li class="page-title-container"><s:text
+            name="%{'title.' + entityManagerName + '.management'}"/></li>
+</ol>
```
Comment (nicpuddu) [inline admin-console/src/main/webapp/WEB-INF/apsadmin/jsp/entity/include/entity-type-list.jsp]: come viene gestito ora l'override del breadcrumb?

**Comment by nicpuddu** [comment]
add to whitelist

**Comment by nicpuddu** [comment]
test this please

**Comment by entando-service** [comment]
test this please

**Comment by entando-service** [comment]
All is well

**Comment by nicpuddu** [comment]
test this please

**Pull request closed without merging**